docker/humble_orbslam3/Dockerfile

```diff
@@ -0,0 +1,90 @@
+FROM osrf/ros:humble-desktop
+
+ARG USERNAME=assume
+
+
+#############################################
+# --- Basic setup ---
+RUN apt update && \
+    apt install sudo vim curl tree -y
+
+# Set the timezone
+RUN DEBIAN_FRONTEND=noninteractive apt install -y --no-install-recommends tzdata
+RUN TZ=Asia/Taipei && \
+    ln -snf /usr/share/zoneinfo/$TZ /etc/localtime && \
+    echo $TZ > /etc/timezone && \
+    dpkg-reconfigure -f noninteractive tzdata 
```

Comment:
Is setting timezone necessary? Since the base image already set the timezone to UTC:

https://github.com/osrf/docker_images/blob/bcc11bef2ba85c2941e66d12a9e9b0a05af0a456/ros/humble/ubuntu/jammy/ros-core/Dockerfile#L5-L10

Setting all nodes to UTC may be better than setting to `Asia/Taipei`?

Reply:
@Tony Guo: Mount local timezone into docker container (in docker-compose) instead